activities/files_test.go

```diff
@@ -0,0 +1,45 @@
+package activities
+
+import (
+	"testing"
+
+	"github.com/bcc-code/bcc-media-flows/paths"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"go.temporal.io/sdk/testsuite"
+)
+
+type UnitTestSuite struct {
+	suite.Suite
+	testsuite.WorkflowTestSuite
+
+	env *testsuite.TestActivityEnvironment
+}
+
+func (s *UnitTestSuite) SetupTest() {
+	s.env = s.NewTestActivityEnvironment()
+}
+
+func (s *UnitTestSuite) TestStandardizeFileName() {
+	p, err := paths.Parse("/mnt/filecatalyst/a  b /c/d e g.txt")
+	t := s.T()
+
+	ua := UtilActivities{}
+	assert.NoError(t, err)
+
+	s.env.RegisterActivity(ua.StandardizeFileName)
+	res, err := s.env.ExecuteActivity(ua.StandardizeFileName, FileInput{Path: p})
+
+	assert.NoError(t, err)
+	assert.NotNil(t, res)
+
+	p2 := &paths.Path{}
+	res.Get(p2)
+
+	assert.NoError(t, err)
+	assert.Equal(t, "/a/b/c/d_e_g.txt", p2.Path)
```

Comment:
I'd expect the spaces in the folder names to become _ as well?

Reply:
Even if the function name is `TestStandardizeFileName`?

Reply:
didnt even think about that. why is it modifying the folder name at all since its standardize**filename**?

either way feels wrong to split each space into a new folder